host-bmc/host_pdr_handler.cpp

```diff
@@ -161,6 +166,14 @@ HostPDRHandler::HostPDRHandler(
                     pldm_entity obj{};
                     this->objPathMap[element.first] = obj;
                 }
+                else if (propVal ==
```

Comment:
this check is already there below can we have the `handleBootTypesAtPowerOn` called in that else if itself? this is a repeatitive else-if